### PR TITLE
Recipe for yapc

### PR DESCRIPTION
**Pull request by jurgjn**
Add a recipe for yapc, a peak caller for high-throughput sequencing data.

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences**
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by jurgjn** [comment]
@BiocondaBot please add label

File: recipes/yapc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+test:
+  commands:
+    - yapc --help > /dev/null
+
+about:
+  home: https://github.com/jurgjn/yapc
+  summary: Yapc is a (yet another) peak caller for genomic high-throughput sequencing data
+  license: GPLv3
+  license_family: GPL
```
Comment (npavlovikj) [inline recipes/yapc/meta.yaml]: Add the license file as well: `license_file: LICENSE`.

File: recipes/yapc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package:
```
Comment (npavlovikj) [inline recipes/yapc/meta.yaml]: I would suggest you to use:
```
{% set name = "yapc" %}
{% set version = "0.1" %}

package:
  name: {{ name|lower }}
  version: {{ version }}
```
so the recipe looks cleaner.

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**